resources/frameworks.yaml

```diff
@@ -69,6 +69,14 @@ AutoWEKA:
   version: '2.6'
   project: https://www.cs.ubc.ca/labs/beta/Projects/autoweka/
 
+MLPlanWEKA:
+  version: 'latest'
+  project: https://mlplan.org
+
+MLPlanSKLearn:
```

Comment:
Just wondering, I didn't compare the 2 integrations line by line, but they look very similar to me. The main difference I see in the `setup` is that `sklearn` version requires of course Python dependencies. 
Wouldn't be easier for future maintenance to have only one integration?
They would then share the same `setup.sh` (downloading sklearn even in Weka mode should be harmless).
And you could just switch between modes in the framework definition, for example:
```yaml
MLPlan:
  module: frameworks.MLPlan
  version: latest
  project: https://mlplan.org

MLPlanWEKA:
    extends: MLPlan
    params:
        _mode: weka

MLPlanSklearn:
    extends: MLPlan
    params:
        _mode: sklearn
```
in the shared `exec.py`, you can then use:
```python
mode = config.framework_params.get('_mode')  # the leading underscore in the param is by convention for params not passed directly to the framework itself
```
to switch some logic between both implementations.

Again, this makes sense only if you think it would simplify integration and avoid duplicate logic.

Reply:
Thanks for the pointer. I will re-arrange the include of the framework definition